### PR TITLE
razzle-plugin-less implemented and tested by copying razzle-plugin-scss

### PR DESCRIPTION
**Pull request by wickedev**
I have read the CONTRIBUTING and CODE OF CONDUCT documents and all package(with-less and razzle-plugin-less) test is passed including Hot reload. Thank you for your review

image

image

### CODE REVIEW

**Comment by wickedev** [comment]
You can also try it out at `yarn add https://github.com/wickedev/razzle-plugin-less#v5.1.0`.

**Comment by wickedev** [comment]
I used for my company internal project with AntD and fixBabelImports and it works fine. Let me know if you need to add or verify anything.

File: examples/with-less/razzle.config.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            sourceMap: true,
+          }),
+          prod: Object.assign(lessOption, {
+            sourceMap: false,
+          }),
+        },
+      },
+    },
+  ],
+};
```
Comment (nimaa77) [inline examples/with-less/razzle.config.js]: just use the default config to keep examples as minimal as possible
```suggestion
module.exports = {
  plugins: ['less'],
};
```

**Review by nimaa77 — COMMENTED** [review]
LGTM, you are awesome 😎

File: examples/with-less/razzle.config.js (hunk 1 above)
Comment (wickedev) [reply examples/with-less/razzle.config.js]: Changed according to suggestion. Thank you.

**Comment by aforty** [comment]
This now works in dev and prod for me now and I've been using this for a few months. Would be great to get this merged into core.

**Review by nimaa77 — APPROVED** [review]

**Comment by nimaa77** [comment]
testing it right now

**Comment by nimaa77** [comment]
awsome, fix the conflict and then we will merge this 👍 
just merge branch `next` to your branch and run `yarn`.
yarn will automatically fix the conflict for you...

File: examples/with-less/src/App.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import './App.less';
+
+import React from 'react';
+
+const App = () => (
+  <div>
+    <div>Welcome to Razzle.</div>
+    <div className="info">Hot reload enabled!!</div>
```
Comment (nimaa77) [inline examples/with-less/src/App.js:8]:
```suggestion
```

**Comment by nimaa77** [comment]
thanks for your contribution and GG 🤛

**Comment by OmarEMC** [comment]
> I used for my company internal project with AntD and fixBabelImports and it works fine. Let me know if you need to add or verify anything.
#1452 <--

**Pull request merged**